src/org/loklak/api/search/TimeAndDateService.java

```diff
@@ -72,4 +81,34 @@ public static SusiThought timeAndDate() {
 		result.setData(jsonArray);
 		return result;
 	}
+
+    private static Date timeToUTC(Date date) {
+
+        SimpleDateFormat dateFormatGmt = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss");
+        dateFormatGmt.setTimeZone(TimeZone.getTimeZone("GMT"));
+
+        //Local time zone   
+        SimpleDateFormat dateFormatLocal = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss");
+
+        try {
+        //Time in GMT
+        date = dateFormatLocal.parse(dateFormatGmt.format(date));
+        
+        } catch (ParseException e) {
+            DAO.severe(e);
+            date = new Date();
+        }
+        return date;
+    }
+
+    private static String timeDiff(Date timeLocal) {
+    
+        long diff = timeLocal.getTime() - timeToUTC(timeLocal).getTime();
+        long diffMin = diff / (60 * 1000) % 60;
+        long diffHrs = diff / (60 * 60 * 1000) % 24;
+        String diffTime = String.valueOf(diffHrs) + ":" + String.valueOf(diffMin).replace("-","");
```

Comment:
@singhpratyush TimeZone.getOffset() and Date.getTime() gives same output, since we are dealing date object, I used .getTime()